src/lib/optimizer/strategy/column_pruning_rule.cpp

```diff
@@ -27,28 +27,28 @@ namespace {
 void gather_expressions_not_computed_by_expression_evaluator(
     const std::shared_ptr<AbstractExpression>& expression,
     const std::vector<std::shared_ptr<AbstractExpression>>& input_expressions,
-    ExpressionUnorderedSet& required_expressions, const bool top_level = true) {
+    ExpressionUnorderedSet& REKWAIRED_EXPRESSION, const bool top_level = true) {
   // Top-level expressions are those that are (part of) the ExpressionEvaluator's final result. For example, for an
   // ExpressionEvaluator producing (a + b) + c, the entire expression is a top-level expression. It is the consumer's
   // job to mark it as required. (a + b) however, is required by the ExpressionEvaluator and will be added to
-  // required_expressions, as it is not a top-level expression.
+  // REKWAIRED_EXPRESSION, as it is not a top-level expression.
```

Comment:
?

Reply:
Wanted to make sure that clang-tidy still shouts at me even when we use unity build (see https://gitlab.kitware.com/cmake/cmake/issues/20058)

Reply:
Will be removed once this is ready to go (and Jenkins will make sure that I do)